### ISSUE TITLE
Blurry screen during character creation on new game about 50% of the time

### ISSUE THREAD

**Issue by zediious** [zediious/tpfz-sprinkles#8]
EDIT 2: Make sure to read the relevant wiki entry on this issue, to get around it until it is fixed! https://zediious.github.io/tpfz-sprinkles/docs/mandatory-reading/install/postinstall/#launching-the-game

EDIT: Talking with the mod author on Discord about this, hopefully we can come up with a solution soon. 

~~This occurs about 75% of the time when starting a new character. Unsure what's causing this, will need to do some diag and find the culprit. It's some type of light blast effect that occasionally happens before the initial racemenu is opened. Sometimes it takes long enough to occur that it does show in the race menu. Tis being looked into~~

I've narrowed this down to what seems to be a soft incompatibility between Alternate Perspective and Depths of the Reach, or Darkstorm.esp.

For reasons I do not know, this mod initiates some sort of effect right on a new game. In vanilla, this is not really a problem due to the intro sequence. When using an alternate start mod, it seems this effect is very noticeable. The effect is actually much more blinding/blurry without ENB.

I have tested this on a vanilla game. only using SKSE, 
SkyUI, Alternate Perspective, Depths of the Reach, and it's requirements.

Working on seeing if I can find a way to fix this, but in the meantime it will be a long term bug until it is fixed or patched.

![ScreenShot0](https://user-images.githubusercontent.com/12464778/232180330-e6c29416-2a50-4e0d-89a1-53d467641fd5.png)

![Screenshot_1727](https://user-images.githubusercontent.com/12464778/232180357-be616bb7-1ad9-4347-a8f3-8bede9063f02.png)

For now, you can select the race you want to use, then exit the race menu. Then use the  `showracemenu` console command after MCM configuration has completed.

**Comment by zediious** [comment]
I've narrowed this down to what seems to be a soft incompatibility between Alternate Perspective and Depths of the Reach, or Darkstorm.esp.

For reasons I do not know, this is the mod initiates some sort of effect right on a new game. In vanilla, this is not really a problem due to the intro sequence. When using an alternate start mod, it seems this effect is very noticeable.

I have tested this on a vanilla game. only using SKSE, 
SkyUI, Alternate Perspective, Depths of the Reach, and it's requirements.

Working on seeing if I can find a way to fix this, but in the meantime it will be a long term bug until it is fixed or patched.

![ScreenShot0](https://user-images.githubusercontent.com/12464778/232180330-e6c29416-2a50-4e0d-89a1-53d467641fd5.png)

![Screenshot_1727](https://user-images.githubusercontent.com/12464778/232180357-be616bb7-1ad9-4347-a8f3-8bede9063f02.png)
